internal/pkg/instrumentation/bpf/go.opentelemetry.io/otel/traceglobal/bpf/probe.bpf.c

```diff
@@ -64,6 +64,14 @@ struct {
 // Injected in init
 volatile const u64 tracer_delegate_pos;
 
+
+// span_name_read reads the span name from the provided span_name_ptr and stores the result in
+// span_name.buf.
+static __always_inline void span_name_read(void *span_name_ptr, struct span_name_t *span_name, const u64 span_name_len) {
```

Comment:
nit: we tend to prefer verb first, so this would be `read_span_name` - not a big issue as this is an internal function though

Reply:
i like that function name/philosophy better. i'll update this later today.

Reply:
updated and resolving thread. please re-open if you have further suggestions :)